web/src/components/apps/AppLicense.tsx

```diff
@@ -157,15 +157,19 @@ const AppLicenseComponent = (props: Props) => {
     const airgapLicense = await yaml.safeLoad(contentStr);
     const { appLicense } = state;
 
-    if (airgapLicense.spec?.licenseID !== appLicense?.id) {
+    // TODO: FIX THIS
+    // @ts-ignore
+    if (airgapLicense?.spec?.licenseID !== appLicense?.id) {
```

Comment:
dunno how this got in. going to standardize the makefiles / linting commands next week so hopefully that will block these kinds of things during PR checks

Reply:
what is the error?

Reply:
the type returned by `TextDecoder` is not the type that `yaml.safeLoad` expects. I'm not sure what any of this is so I'm avoiding changing it for now. 

```
 const content: any = await getFileContent(files[0]);
    const contentStr = new TextDecoder("utf-8").decode(content);
    const airgapLicense = await yaml.safeLoad(contentStr);
    const { appLicense } = state;

    if (airgapLicense.spec?.licenseID !== appLicense?.id) {
    // TODO: FIX THIS
    // @ts-ignore
    if (airgapLicense?.spec?.licenseID !== appLicense?.id) {
```